Sources/nest/Commands/RunCommand.swift

```diff
@@ -0,0 +1,174 @@
+import ArgumentParser
+import Foundation
+import NestKit
+import NestCLI
+import Logging
+
+struct RunCommand: AsyncParsableCommand {
```

Comment:
`nest` module should be thin as much as possible.
So could you try to move the logic to other modules and reduce the logic in this module?

Reply:
Thanks!
I tried to corrected here. (https://github.com/mtj0928/nest/pull/36/commits/1ee1a5f6d48e2a0a025a28c9b5ba50574e2e38b6, https://github.com/mtj0928/nest/pull/36/commits/8306591f62f77a728946a8859876a2966b324748)